Scenes/Screens/RevivalShop/RevivalShop.tscn

```diff
@@ -3,450 +3,462 @@
 [ext_resource path="res://Scenes/Screens/RevivalShop/RevivalShop.gd" type="Script" id=1]
```

Comment:
[I am commenting this file, not this line] You did this partially correct. The rows (items) in the scene should not be copy-paste but should be instances of an individual scene (for example called: `RevivalItem.tscn`). That makes scene tree more professional and much more flexible. To change the colour of `Count` label you can use `exporterd variable` and tool mode to make it wokr in real time in editor: https://docs.godotengine.org/en/3.1/tutorials/misc/running_code_in_the_editor.html.